### PR TITLE
mongodb 3.0.5

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/mongodb.rb [hunk 1]
```diff
@@ -5,16 +5,25 @@ class Mongodb < Formula
...
   end
 
+  bottle do
+    cellar :any
+    sha256 "cb796f8f0aaa0f457660d6cfca16325d4a708ffe21e66cba7caabb7e58932f7d" => :yosemite
+    sha256 "5a5ad8cb4399a08bcc4dd05297d29bfaf225676d58df3f4c7d85c0e4b223df81" => :mavericks
+    sha256 "e0bfd72074a21e1090632018d4f91ea3f89d2694a8189bc6d1af7e2dfb7f608e" => :mountain_lion
+  end
+
+  needs :cxx11
```
Comment (dunn) [inline Library/Formula/mongodb.rb]: This should probably be down with the `depends_on` lines, so it's harder to overlook.

**Comment by xu-cheng** [comment]
Thanks @nijikon

**Comment by anthonybrown** [comment]
I am unable to update to monogdb-3.0.5, this is the error:
```
/usr/bin/clang++ -o build/darwin/cc__usr_bin_clang/cxx__usr_bin_clang++/osx-version-min_10.10/ssl/use-system-boost/mongo/shell/linenoise.o -c -Wnon-virtual-dtor -Woverloaded-virtual -fPIC -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch -pipe -O3 -Wno-unused-function -Wno-unused-private-field -Wno-deprecated-declarations -Wno-tautological-constant-out-of-range-compare -Wno-unused-const-variable -Wno-missing-braces -mmacosx-version-min=10.10 -D_SCONS -DMONGO_EXPOSE_MACROS -DPCRE_STATIC -DSUPPORT_UTF8 -DMONGO_OPTIMIZED_BUILD -DMONGO_BYTE_ORDER=1234 -D_FILE_OFFSET_BITS=64 -DMONGO_SSL -DMONGO_HAVE___THREAD -DMONGO_HAVE_GCC_ATOMIC_BUILTINS -DBOOST_SYSTEM_NO_DEPRECATED -DBOOST_THREAD_VERSION=2 -DMONGO_HAVE_HEADER_UNISTD_H -DMONGO_HAVE_EXECINFO_BACKTRACE -DMONGO_HAVE_FIPS_MODE_SET -I/usr/local/opt/openssl/include -Isrc/third_party/s2 -Isrc/third_party/pcre-8.37 -Ibuild/darwin/cc__usr_bin_clang/cxx__usr_bin_clang++/osx-version-min_10.10/ssl/use-system-boost -Isrc src/mongo/shell/linenoise.cpp
/usr/bin/clang++ -o build/darwin/cc__usr_bin_clang/cxx__usr_bin_clang++/osx-version-min_10.10/ssl/use-system-boost/mongo/shell/linenoise_utf8.o -c -Wnon-virtual-dtor -Woverloaded-virtual -fPIC -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch -pipe -O3 -Wno-unused-function -Wno-unused-private-field -Wno-deprecated-declarations -Wno-tautological-constant-out-of-range-compare -Wno-unused-const-variable -Wno-missing-braces -mmacosx-version-min=10.10 -D_SCONS -DMONGO_EXPOSE_MACROS -DPCRE_STATIC -DSUPPORT_UTF8 -DMONGO_OPTIMIZED_BUILD -DMONGO_BYTE_ORDER=1234 -D_FILE_OFFSET_BITS=64 -DMONGO_SSL -DMONGO_HAVE___THREAD -DMONGO_HAVE_GCC_ATOMIC_BUILTINS -DBOOST_SYSTEM_NO_DEPRECATED -DBOOST_THREAD_VERSION=2 -DMONGO_HAVE_HEADER_UNISTD_H -DMONGO_HAVE_EXECINFO_BACKTRACE -DMONGO_HAVE_FIPS_MODE_SET -I/usr/local/opt/openssl/include -Isrc/third_party/s2 -Isrc/third_party/pcre-8.37 -Ibuild/darwin/cc__usr_bin_clang/cxx__usr_bin_clang++/osx-version-min_10.10/ssl/use-system-boost -Isrc src/mongo/shell/linenoise_utf8.cpp
2 warnings and 1 error generated.
scons: *** [build/darwin/cc__usr_bin_clang/cxx__usr_bin_clang++/osx-version-min_10.10/ssl/use-system-boost/mongo/shell/bench.o] Error 1
scons: building terminated because of errors.
```

**Comment by nijikon** [comment]
@anthonybrown sorry for that, can you provide us with ```brew config```, thanks!

**Comment by anthonybrown** [comment]
@nijikon ```zsh: correct 'config' to '.config' [nyae]? n
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew
HEAD: 4a2e55198160bee6b3d741fc726f777e103f16cf
Last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: 8-core 64-bit sandybridge
OS X: 10.10.4-x86_64
Xcode: 6.4
CLT: 6.4.0.0.1.1435007323
Clang: 6.1 build 602
X11: 2.7.7 => /opt/X11
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /Users/tony/.rvm/rubies/ruby-2.2.2/bin/ruby
Java: 1.6.0_65```

**Comment by anthonybrown** [comment]
@nijikon `3.0.5` builds fine on my mavericks machine but not on yosemite

**Comment by anthonybrown** [comment]
Here is my (gist)[https://gist.github.com/d119a960afbecfc247a1]

**Comment by anthonybrown** [comment]
Basically mongodb will not build: `--with-openssl`

**Comment by dunn** [comment]
The gist you posted was `--with-openssl` and `--with-boost`, it looks like?

**Comment by anthonybrown** [comment]
@dunn correct, that's why it's not building

**Comment by dunn** [comment]
Interesting; so it builds fine `--with-openssl` and builds with `--with-boost`, but not with both together?

**Comment by anthonybrown** [comment]
It does not build: `--with-openssl` or `--with-boost` 
I tried it both ways, it doesn't build if you add openssl or boost but will build with them

**Comment by dunn** [comment]
Oh, got it.

**Comment by dunn** [comment]
`brew install -fs mongodb --with-openssl` works for me, with this setup:
```
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew.git
HEAD: 4d5a2951aab0296b244a89b9b9b9591a91f9b165
Last commit: 20 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
OS X: 10.10.4-x86_64
Xcode: 6.4
CLT: 6.4.0.0.1.1435007323
Clang: 6.1 build 602
X11: 2.7.7 => /opt/X11
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/local/bin/ruby => /usr/local/Cellar/ruby/2.2.2/bin/ruby
Java: 1.8.0_20
```

**Comment by dunn** [comment]
The `--with-boost` failure is known: https://jira.mongodb.org/browse/SERVER-19661

**Comment by anthonybrown** [comment]
@dunn thanks, been using the mongo install without boost, would still like to find a solution though :+1:

**Comment by dunn** [comment]
@anthonybrown try this:
```
brew reinstall boost --c++11
brew install mongodb --with-boost
```

**Pull request closed without merging**